LSLib/LS/FileManager.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.IO;
+
+namespace LSLib.LS
+{
+    internal class FileManager
+    {
+        public static bool TryToCreateDirectory(string path)
```

Comment:
I think we should hard fail here instead of returning a bool. The return code is not checked anywhere, so at the moment it is useless. Also, if the directory creation fails it is quite obvious that export file creation will also fail, since the directory it'll write to is not present.

Reply:
By hard fail, do you mean throwing exceptions, or just returning silently? Preference?

This is LSLib so I'm assuming MessageBoxes are out.

Reply:
An exception would be the best I think.